lms/djangoapps/branding/tests/test_page.py

```diff
@@ -295,7 +295,7 @@ class IndexPageProgramsTests(SiteMixin, ModuleStoreTestCase):
     """
     def test_get_programs_with_type_called(self):
         views = [
-            (reverse('root'), 'student.views.get_programs_with_type'),
+            (reverse('root'), 'student.views.management.get_programs_with_type'),
```

Comment:
This code moved a while ago, but we were using an assertion not implemented in mock 1.0.1 so the test kept passing.